llm/gguf.go

```diff
@@ -5,12 +5,19 @@ import (
 	"encoding/binary"
 	"fmt"
 	"io"
+	"os"
+	"regexp"
+
+	"github.com/d4l3k/go-bfloat16"
+	"github.com/pdevine/tensor"
+	"github.com/pdevine/tensor/native"
+	"github.com/x448/float16"
 
 	"github.com/jmorganca/ollama/format"
 )
 
-type containerGGUF struct {
-	bo binary.ByteOrder
+type ContainerGGUF struct {
```

Comment:
I don't think these needs to be exported. Instead, convert should be moved into this package